acvp_transport.c

```diff
@@ -54,12 +54,14 @@ static long acvp_curl_http_get (ACVP_CTX *ctx, char *url, void *writefunc)
     long http_code = 0;
     CURL *hnd;
     struct curl_slist *slist;
-    char bearer[1048]; //TODO: remove this from the stack
 
     slist = NULL;
     if (ctx->jwt_token) {
-        snprintf(bearer, 1048, "Authorization: Bearer %s", ctx->jwt_token);
+    	int buf_size = strlen(ctx->jwt_token) + 22;
```

Comment:
We should probably use strnlen() instead of strlen(), then use a #define to declare a maximum JWT length that we'll allow.  Keep in mind the JWT is coming off the wire from the server.  It's best to never trust data coming off the wire.